### ISSUE TITLE
Angular v12 throwing Must call super constructor in derived class before accessing 'this' or returning from derived constructor error

### ISSUE THREAD

**Issue by codef0rmerz** [easepick/easepick#85]
**Describe the bug**
In Angular v12+, it's throwing `Must call super constructor in derived class before accessing 'this' or returning from derived constructor` error.

```
new easepick.create({
      element: this.document.getElementById('foo') as HTMLInputElement,
      css: [
        'https://cdn.jsdelivr.net/npm/@easepick/core/dist/index.css',
        'https://cdn.jsdelivr.net/npm/@easepick/range-plugin/dist/index.css',
      ],
})
```

**To Reproduce**
1. Generate Angular v12 application and run it on Prod mode i.e. optimization: true in angular.json
2. Add an input tag in app.component.html with an id `foo`
3. Call the above easepick.create method in the constructor of the component class
4. You will see the error.

**Expected behavior**
Turns out, [in the source code](https://github.com/easepick/easepick/blob/master/packages/datetime/src/index.ts#L223), this `;` (separating the super method and this.lang assignment) turns into `,` at build times.

**Code Playground**
N/A. 

**Additional context**
Angular v12

**Comment by wakirin** [comment]
Angular v12: https://angular-ivy-z8et9p.stackblitz.io
[edit link](https://stackblitz.com/edit/angular-ivy-z8et9p?file=src/app/app.component.ts)

Angular v14: https://angular-ivy-y2wzcn.stackblitz.io/ 
[edit link](https://stackblitz.com/edit/angular-ivy-y2wzcn?file=src/app/app.component.ts)

Probably this issue depends on something, but I don't this problem on clean angular app.
Please provide a link to the code playground with this problem to re-open the issue.

**Comment by codef0rmerz** [comment]
@wakirin I could not get to the source code of your playgrounds so can you check if you have enabled buildOptimiser and optimization ON in angular.json. Sometimes, it's OFF in dev mode.

**Comment by dalmenarDev** [comment]
I have the same problem in Angular 13 with optimization: true. If I set optimization: false, it's working. but... I don't know what happen because I'm calling library as it says in the documentation

`
import {  easepick } from '@easepick/bundle'


 this.datePicker = new easepick.create(datePickConfig)
`

**Comment by nettags** [comment]
I have also the same issue with Angular 13 (latest 13 version) with optimization:true.

To make sure that the problem is not related to my code I have set up a fresh Angular 13 installation using CLI (ng new). Here I copied the Stackblitz code from @wakirin and I'm able to reproduce the error. By default "optimization" is set to false in the development environment (or ng serve), the error occurs only after building with "ng build".

@wakirin - if you build your Stackblitz code, you should be able to see the error in the browser - its not shown during building.

Unfortunately, using optimization:false is not possible for me because of the bundle/file sizes - the simple example code with just the datepicker has a size of 1.63 MB instead of 153.12 kB with optimization:true.

@wakirin please re-open this issue and let me/us know if you need any assistance.

**Comment by nettags** [comment]
I did a little more research. After updating via CLI to Angular 14 the error is gone, even after building with optimization:true. During the update the compilerOptions.target in tsconfig.json is set to "es2020" on Angular 14, on Angular 13 the target was at "es2017" - if I manually set the target on Angular 14 back to "es2017" then the error is also back.

I hope this info helps you for better understanding or fixing this issue in your Angular project.
